libbeat/docs/filteringconfig.asciidoc

```diff
@@ -0,0 +1,63 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/filteringconfig.asciidoc[]
+//// Make sure this content appears below a level 2 heading.
+//////////////////////////////////////////////////////////////////////////
+
+[[configuration-filter]]
+=== Filter
+
+You can set options in the `filter` section of the {beatname_uc} config file to reduce the number of fields that are 
+exported by the Beat. See <<exported-fields>> for the full list of possible fields.
+
+The filter actions that you specify are applied to each event before it's sent to the defined output. You can define
+multiple filter actions. The filter actions are executed in the order that they're defined in the config file.
+
+==== Filter Options
+
+You can specify the following options under the `filter` section:
+
+[[include-fields]]
+===== include_fields
+
+The `include_fields` action specifies the list of fields to export. You can define full nested maps as fields. 
+
+Example configuration:
+
+[source, yaml]
+-----------------------------------------------------
+filter:
+ - include_fields:
+     fields: ["field_A", "field_B", "field_C"]
+-----------------------------------------------------
+
+If `include_fields` is not specified, all fields are exported by default. 
+
+See <<filtering-data>> for specific {beatname_uc} examples.
+
+NOTE: If you specify an empty list for `include_fields`, only required fields are exported. To find out which
```

Comment:
@monicasarbu Given Andrew's point above, I think we need to clarify what we mean by "required" in the doc about output fields. Because now we are talking about two different categories of required.